src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.General.Self": "Self: Contrôlez le temps du monde de jeu en ignorant les mises à jour des autres modules.",
+  "FSC.Configuration.General.ThirdParty": "Module tiers : Suit uniquement les mises à jour du temps du monde de jeu provenant d'autres modules.",
+  "FSC.Configuration.General.Mixed": "Hybride : Mettre à jour le temps du monde de jeu et suivre les mises à jour effectuées à partir d'autres modules.",
+  "FSC.Configuration.General.PF2ESync": "Pathfinder 2E : synchronisation du temps du monde de jeu",
+  "FSC.Configuration.General.PF2ESyncHelp": "Cela forcera Simple Calendar à rester synchronisé avec le temps du monde de jeu intégrée de Pathfinder 2E.",
+  "FSC.Configuration.General.ShowClock": "Afficher l'horloge",
+  "FSC.Configuration.General.ShowClockHelp": "Si pour afficher l'horloge sous le calendrier, si coché, les commandes pour changer les heures, les minutes et les secondes s'afficheront également.",
+  "FSC.Configuration.General.PlayerAddNotes": "Ajouter des notes",
+  "FSC.Configuration.General.PlayerAddNotesHelp": "Les rôles cochés pourront ajouter des notes au calendrier.",
+  "FSC.Configuration.General.PlayerReorderNotes": "Réorganiser les notes",
+  "FSC.Configuration.General.PlayerReorderNotesHelp": "Les rôles cochés pourront réorganiser les notes un jour.",
+  "FSC.Configuration.General.QuickSetup": "Configuration rapide",
+  "FSC.Configuration.General.Notes": "Notes",
+  "FSC.Configuration.General.ImportExport.Title": "Importer/Exporter",
+  "FSC.Configuration.General.ImportExport.SCConfig": "Importer/Exporter la configuration du calendrier",
+  "FSC.Configuration.General.ImportExport.SCConfigHelp": "Pour importer ou exporter la configuration actuelle du calendrier Simple Calendar pour partager ou charger la réplication dans un autre monde.",
+  "FSC.Configuration.General.ThirdPartyModule": "Importation/Exportation de modules tiers",
+  "FSC.Configuration.General.Permissions.Note": "Ici, les autorisations pour diverses interactions avec Simple Calendar peuvent être ajustées en fonction des rôles de Foundry. Les maîtres de jeu auront toujours ces autorisations.",
+  "FSC.Configuration.General.CalendarPlayerVisible": "Afficher le calendrier",
+  "FSC.Configuration.General.CalendarPlayerVisibleHelp": "Les rôles cochés pourront ouvrir Simple Calendar.",
+  "FSC.Configuration.General.PlayersChangeDateTime": "Modifier la date et l'heure",
+  "FSC.Configuration.General.PlayersChangeDateTimeHelp": "Les rôles cochés pourront changer la date et l'heure de Simple Calendar",
+  "FSC.Configuration.Notes.Title": "Paramètres des notes",
+  "FSC.Configuration.Notes.NoteCategories": "Catégories de notes",
+  "FSC.Configuration.NoteCategories.Name": "Nom de la catégorie de note",
+  "FSC.Configuration.NoteCategories.NameHelp": "Le nom d'affichage de cette catégorie de note.",
+  "FSC.Configuration.NoteCategories.Color": "Note Catégorie Couleur",
```

Comment:
This should be "Couleur de la catégorie de notes". This version does not make sense.